sshcode.go

```diff
@@ -140,13 +144,19 @@ func sshCode(host, dir string, o options) error {
 		flog.Info("synced extensions in %s", time.Since(start))
 	}
 
+	codeServerFlags := []string{
+		fmt.Sprintf("--bind-addr 127.0.0.1:%v", o.remotePort),
+		"--auth none",
+	}
+	codeServerCmdStr := fmt.Sprintf("%v/code-server %v %v", codeServerDir, dir, strings.Join(codeServerFlags, " "))
```

Comment:
This change is mostly a groundwork for https://github.com/cdr/sshcode/pull/178 which I will resubmit (with additional changes I've discovered necessary) after this PR get merged.